### DISCUSSION TITLE
Rename codespaces

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#17101] [Codespaces]
It would be cool to be able to change the name of a codespace

**Comment by legomushroom** [comment]
@Milo123459 thanks for the feedback 👍 

cc @nikmd23 @tanmayeekamath @craiglpeters do you know if we going to support this soon?

**Reply by floriangrousset** [reply]
@legomushroom @nikmd23 @tanmayeekamath @craiglpeters – Any visibility on if/when you'd add that feature? Thank you!

**Comment by AyushSehrawat** [comment]
Oh, codespaces has no feature to rename currently ? Well, i guess have to wait until they do

**Comment by msimecek** [comment]
Any news on this, please? I have codespaces based on different configurations and there's no way to tell them apart (or at least I don't see it)... Being able to specify a name, or at least part of it, would be really handy.

![image](https://user-images.githubusercontent.com/9624293/184380972-6f2cbff9-0603-47d9-b59b-d122b47d3c6d.png)

**Comment by thomas-sickert** [comment]
This functionality has been added in the [GitHub CLI](https://cli.github.com/manual/gh_codespace_edit). Unfortunately, I don't have an estimate on support for this feature in other clients/github.com, but hopefully this will help for now.

**Reply by AyushSehrawat** [reply]
Thanks @thomas-sickert for the info 🙂!
